website/docs/reference/resource-configs/snapshot_meta_column_names.md

```diff
@@ -0,0 +1,109 @@
+---
+resource_types: [snapshots]
+description: "Snapshot meta column names"
+datatype: "{<dictionary>}"
+default_value: {"dbt_valid_from": "dbt_valid_from", "dbt_valid_to": "dbt_valid_to", "dbt_scd_id": "dbt_scd_id", "dbt_updated_at": "dbt_updated_at"}
+id: "snapshot_meta_column_names"
+---
+
+Available in v1.9 or with [versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) dbt Cloud.
```

Comment:
starting in 1.9?

Reply:
do we want to make this page versioned? (only be available for 1.9+)

Reply:
<img width="661" alt="Screenshot 2024-10-02 at 4 51 48 PM" src="https://github.com/user-attachments/assets/7fc7baae-15b1-434c-9e70-d4021ddb5e12">

nvm i see that you've already done this!

Reply:
I did it based on your suggestion -- good call 🙌